docker/1.4.0/final/Dockerfile.gpu

```diff
@@ -3,6 +3,7 @@ ARG py_version
 FROM awsdeeplearningteam/mxnet-model-server:base-gpu-py$py_version
 
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```

Comment:
Multi-model mode is only supported in CPU containers

Reply:
Removed